src/hokusai/hokusai.yml

```
@@ -47,9 +47,12 @@ jobs:
         default: 20m
     steps:
       - setup
+      - run:
+          name: Install AWS IAM Authenticator
+          command: curl -o aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/1.13.7/2019-06-11/bin/linux/amd64/aws-iam-authenticator && chmod +x ./aws-iam-authenticator && mv aws-iam-authenticator /usr/local/bin/
```

I explain this more below, but it's really just for legibility

```suggestion
          command: |
            curl -o aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/1.13.7/2019-06-11/bin/linux/amd64/aws-iam-authenticator 
            chmod +x ./aws-iam-authenticator 
            mv aws-iam-authenticator /usr/local/bin/
```